Error building wazuh[curl] on macOS x86

## Description
After the upgrade to DEPS 35, we receive this error during the build process in macOS x64.

```
configure: error: /Users/***/Documents/wazuh/src/external/openssl/ is a bad --with***penssl prefix!
make[2]: *** [config.status] Error 1
make[1]: *** [../config.status] Error 2
make: *** [external/curl/lib/.libs/libcurl.a] Error 2
```
After that we decide to rebuild the curl 8.10.1 and during the compilation on macOS Sierra we receive an other error:

```
checking for psl_builtin in -lpsl... no
configure: error: libpsl libs and/or directories were not found where specified!
make: *** [external/curl/Makefile] Error 1
```


## DoD
- [x] Fix libpsl error in the compilation machine.
- [x] Upload dependencies to s3

--- a/src/Makefile
+++ b/src/Makefile
@@ -1168,7 +1168,7 @@ else
 	cd $(EXTERNAL_CURL) && CPPFLAGS="-fPIC -I${ROUTE_PATH}/${EXTERNAL_OPENSSL}include" LDFLAGS="-L${ROUTE_PATH}/${EXTERNAL_OPENSSL}" LIBS="-ldl -lpthread" ./configure --with-openssl="${ROUTE_PATH}/${EXTERNAL_OPENSSL}" --disable-ldap --without-libidn2 --without-libpsl --without-brotli --without-nghttp2 --without-zstd
 endif
 else
-	cd $(EXTERNAL_CURL) && CPPFLAGS="-fPIC -I${ROUTE_PATH}/${EXTERNAL_OPENSSL}include" LDFLAGS="-L${ROUTE_PATH}/${EXTERNAL_OPENSSL}" LIBS="-lpthread" ./configure --with-openssl="${ROUTE_PATH}/${EXTERNAL_OPENSSL}" --disable-ldap --without-libidn2 --without-brotli --without-nghttp2 --without-librtmp --without-zstd
+	cd $(EXTERNAL_CURL) && CPPFLAGS="-fPIC -I${ROUTE_PATH}/${EXTERNAL_OPENSSL}include" LDFLAGS="-L${ROUTE_PATH}/${EXTERNAL_OPENSSL}" LIBS="-lpthread" ./configure --with-openssl="${ROUTE_PATH}/${EXTERNAL_OPENSSL}" --disable-ldap --without-libidn2 --without-brotli --without-nghttp2 --without-librtmp --without-zstd --without-libpsl
 endif
 endif
 

